src/components/forums/topic/PostContent.tsx

```diff
@@ -364,13 +376,19 @@ export function PostContent(props: PostContentProps) {
                     )}
                   </div>
                   <div className="walletId">
-                    {identity ? identity.displayName : post.poster.toBase58()}
-                    <RoleLabel
-                      topicOwnerId={topicPosterId}
-                      posterId={post?.poster}
-                      moderators={participatingModerators}
-                    />
+                    {identity
+                      ? identity.displayName
+                      : windowSize < 768
```

Comment:
are we able to do this with CSS instead with just using a wrap or overflow property, rather than just having it with one screen size in js

Reply:
If I use only css, I cannot show the last 4 characters of he wallet. The wallet will be trimmed and followed by an ellipsis, like the image below. Is that a better option?
![image](https://user-images.githubusercontent.com/47254484/190648236-27795468-4362-43dc-8162-e8a39458e781.png)